common/blockchain/src/block.rs

```diff
@@ -42,10 +40,81 @@ pub struct Block<PK> {
     /// very next block will use current’s block hash as `epoch_id`.
     pub epoch_id: CryptoHash,
 
+    /// If present, hash of the serialised epoch *the next* block will belong
```

Comment:
when would it be present, and when wouldn't it?

Reply:
added more docs